contracts/coordinator/src/state.rs

```diff
@@ -220,12 +217,22 @@ pub fn save_deployed_contracts(
 
 pub fn deployed_contracts(
```

Comment:
maybe better to call this deployments or deployed_chains (everywhere this is relevant), because a single entry already has multiple deployed contracts, so it's hard to differentiate between these cases